bower.json

```diff
@@ -22,7 +21,7 @@
   ],
   "license": "Apache License, v2.0",
   "dependencies": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
```

Comment:
This reverts changes someone else sent via PR, the ~ keeps it at 3.3.x but the ^ allows 3.x.x which is more appropriate.